ui/pages/Project/components/edit-families-and-individuals/BulkEditForm.jsx

```diff
@@ -1,143 +1,33 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import { connect } from 'react-redux'
-import { Table } from 'semantic-ui-react'
 
 import { getCurrentProject, getUser } from 'redux/selectors'
-import { FileLink } from 'shared/components/buttons/ExportTableButton'
-import FileUploadField from 'shared/components/form/XHRUploaderField'
 import ReduxFormWrapper from 'shared/components/form/ReduxFormWrapper'
-import { NoBorderTable } from 'shared/components/StyledComponents'
-import { INDIVIDUAL_HPO_EXPORT_DATA, FILE_FIELD_NAME } from 'shared/utils/constants'
+import BaseBulkContent from 'shared/components/form/BulkUploadForm'
```

Comment:
you should import this using the filename unless there is a very good reason not to, so this should be `BulkUploadForm`

Reply:
`BaseBulkContent` is going to be connected in this file. So, it is a real `base` component. Don't you think it is a good reason to keep the name in this file?

Reply:
no, I think its confusing to rename an imported component